integration_tests/models/spark.yml

```diff
@@ -0,0 +1,74 @@
+version: 2
+
+sources:
+  - name: spark_external
+    schema: "{{ target.schema }}"
+    loader: S3
+
+    tables:
+      
+      - name: people_csv_unpartitioned_using
+        external: &csv-people-using
+          location: "s3://dbt-external-tables-testing/csv/"
+          using: csv
+          options: &csv-people-options
```

Comment:
@jtcohen6 What does the &  in front of these do?

Reply:
This looks like some fancy pointer or something.

Reply:
YAML anchors! They're so good. No great docs online, some ok examples [here](https://docs.gitlab.com/ee/ci/yaml/#anchors) and [here](https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/)